### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by yujingxin** [RedMediaTech/ODTSR#8]
I tried training from scratch using the default configuration but without text prompt. How significant is the impact of not using text annotations? Training without prompts results in outputs that are somewhat similar but exhibit reduced sharpness and clarity.

**Comment by Feynman1999** [comment]
> I tried training from scratch using the default configuration but without text prompt. How significant is the impact of not using text annotations? Training without prompts results in outputs that are somewhat similar but exhibit reduced sharpness and clarity.

Could you provide more details about the training setup?

1. Are you currently using empty prompts for both the generator and the discriminator?

2. Is the training dataset the same as the one used in the paper?

3. After training, did you run inference and compare the results with the official checkpoint to reach this conclusion?

**Comment by yujingxin** [comment]
thank u for replying.
1. yes. use empty prompts for both the generator and the discriminator
2. yes. same as the one used in the paper
3. yes. also test with the official checkpoint

**Comment by yujingxin** [comment]
@Feynman1999 Additionally, I’d like to ask another question: I fine-tuned the model with the official checkpoint, performed just one gradient update, saved the model immediately, and then ran an evaluation. I noticed that the deblurring performance dropped significantly—and was quite similar to the results I got when training from scratch. Do you have any suggestions regarding this issue?

**Comment by Feynman1999** [comment]
1. Based on the information you provided, my understanding is that the performance difference is most likely caused by training with empty prompts. In the paper, we indeed did not include an ablation study analyzing the effect of setting all prompts to empty during training.

2. In the future, we will organize and release our training dataset (512×512 crops from LSDir/FFHQ, along with the corresponding prompts) to help the community reproduce the training process.

3. Regarding your comment about loading the model and evaluating the result after only one iteration, the outcome is mainly affected by two factors:
1️⃣ Since this is adversarial training, the discriminator is in a poor initial state, which negatively impacts the generation quality.
2️⃣ If the corresponding optimizer states are not loaded, the performance will also be affected; typically, it requires dozens to hundreds of iterations before the results start to improve.

**Comment by yujingxin** [comment]
Thank you for your reply, and I look forward to further updates on your work.

**Comment by yujingxin** [comment]
@Feynman1999 By the way, regarding the reproduction experiments, could you please help provide the following two text files?
lsdir/lsdir_range1_0_3_range2_1_84991.txt
ffhq_pre_1w/ffhq_pre1w_range1_0_3_range2_0_9999.txt

**Comment by Feynman1999** [comment]
> [@Feynman1999](https://github.com/Feynman1999) By the way, regarding the reproduction experiments, could you please help provide the following two text files? lsdir/lsdir_range1_0_3_range2_1_84991.txt ffhq_pre_1w/ffhq_pre1w_range1_0_3_range2_0_9999.txt

The format of the .txt file is as follows. The dataloader reads images (cropped to 512×512) line by line, and according to a predefined rule, replaces the /img/ part of the path with /prompt/ to load the preprocessed prompts.

for example:
xxxxxx/lsdir/0/img/0000001.png
xxxxxx/lsdir/0/prompt/0000001.txt

<img width="1002" height="250" alt="Image" src="https://github.com/user-attachments/assets/6e76318b-49ca-4799-8f8b-544adc958f8a" />

By the end of January, we will upload the dataset used for training, including the cropped images and their corresponding prompts.

**Comment by llstela** [comment]
> > [@Feynman1999](https://github.com/Feynman1999) By the way, regarding the reproduction experiments, could you please help provide the following two text files? lsdir/lsdir_range1_0_3_range2_1_84991.txt ffhq_pre_1w/ffhq_pre1w_range1_0_3_range2_0_9999.txt
> 
> The format of the .txt file is as follows. The dataloader reads images (cropped to 512×512) line by line, and according to a predefined rule, replaces the /img/ part of the path with /prompt/ to load the preprocessed prompts.
> 
> for example: xxxxxx/lsdir/0/img/0000001.png xxxxxx/lsdir/0/prompt/0000001.txt
> 
> <img alt="Image" width="1002" height="250" src="https://private-user-images.githubusercontent.com/25549636/536550731-6e76318b-49ca-4799-8f8b-544adc958f8a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzYxNzI4NTcsIm5iZiI6MTc3NjE3MjU1NywicGF0aCI6Ii8yNTU0OTYzNi81MzY1NTA3MzEtNmU3NjMxOGItNDljYS00Nzk5LThmOGItNTQ0YWRjOTU4ZjhhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDE0VDEzMTU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjYjI1MzMzNmY1ZDc3OGZhOWRjODUzY2Y1MjM3ZGM1NzUxOTgzNGEyYWQ0M2FlNTZkNDhlMzY4NDAxMWFlNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.1S0djSK7Z2Q2vLejRMx-oTRCr-5l2hnPG-XoF1hjZkM">
> By the end of January, we will upload the dataset used for training, including the cropped images and their corresponding prompts.

你好，请问还有计划开源训练数据预处理（随机crop和Qwen-VL的文本标注）相关代码吗？
